packages/v4/patternfly-docs/content/design-guidelines/styles/colors/colors.md

```diff
@@ -163,16 +160,16 @@ import './colors.css';
 <Grid hasGutter>
```

Comment:
fwiw I don't know if we want to document it, but we have top, right, bottom, and left versions of all of these shadow sizes to create a shadow on that side of the box.